libkbfs/data_types.go

```diff
@@ -539,6 +539,11 @@ func (et EntryType) String() string {
 	return "<invalid EntryType>"
 }
 
+// IsFile returns whether or not this entry points to a file.
+func (et EntryType) IsFile() bool {
```

Comment:
This is unused in this file -- will this be used in a future PR?

(FWIW, there are a bunch of places that could use this helper function.)